SwaMe/FileMaker.cs

```diff
@@ -40,12 +41,13 @@ public FileMaker(int division, string inputFilePath, MzmlParser.Run run, SwathGr
             this.MS2Density50 = MS2Density50;
             this.MS2DensityIQR = MS2DensityIQR;
             this.MS1Count = MS1Count;
+            this.date = date;
         }
 
         public void MakeMetricsPerSwathFile(SwathGrouper.SwathMetrics swathMetrics,string inputFile)
         {
             //tsv
-            StreamWriter streamWriter = new StreamWriter("MetricsBySwath_"+ run.SourceFileName + ".tsv");
+            StreamWriter streamWriter = new StreamWriter(date +"_MetricsBySwath_" + run.SourceFileName + ".tsv");

```

Comment:
Use String.Format to build string - leave these for now because I expect to rebuild this whole class